bin/dar

```diff
@@ -265,6 +265,24 @@ END
 
   Process::Status->assert_ok("❌ Fixing git permissions in container");
 
+  my $config_file = path('~/.cyrus-docker/config');
+  if (-e $config_file) {
+    run(
+      [
+        'docker', 'cp', '--quiet',
+        $config_file->absolute,
+        "$container_id:/etc/cyrus-docker.json",
+      ],
+      _emptyref(),
+      _emptyref(),
+    );
```

Comment:
This should not fail, but if it does I think the two emptyrefs() mean we won't provide any useful output to the user as to why it failed? Should we include any output?

Reply:
Nah, it goes:

```perl
run \@cmd, $input, $output, $errput;
```

We're saying "no input, ignore output", but stderr is still printed.